pkg/metrics/scheduler.go

```diff
@@ -265,6 +274,12 @@ func (m *SchedulerMetrics) SetTotalApplicationsRunning(value int) {
 	m.totalApplicationsRunning.Set(float64(value))
 }
 
+func (m *SchedulerMetrics) getTotalApplicationsRunning() (int, error) {
+	metric := &dto.Metric{}
+	err := m.totalApplicationsRunning.Write(metric)
+	return int(*metric.Gauge.Value), err
+}
```

Comment:
See `getAllocatedContainers` same nil pointer here, solution is same, don't use named returns